CMakeLists.txt

```diff
@@ -1,7 +1,9 @@
 cmake_minimum_required(VERSION 3.23)
 project(soar
   VERSION 9.6.3
-  LANGUAGES CXX)
+  LANGUAGES CXX C)
```

Comment:
What is C used for? I thought it was used in the viewer but not in the rest of SVS, but the viewer is going away. Or is it in a dependency?